admin/settingstree.php

```diff
@@ -0,0 +1,45 @@
+<?php
+// must be run within Dokuwiki
+if(!defined('DOKU_INC')) die();
+
+/**
+ * All DokuWiki plugins to extend the admin function
+ * need to inherit from this class
+ */
+class admin_plugin_dw2pdf_settingstree extends DokuWiki_Admin_Plugin {
+	private $helper = null;
```

Comment:
If you (as one of the core-dev team) say that the settingstree should be refactored to be an only-extended confmanager, then I will do that (time...). 

__BUT__ that's a dokuwiki design concept question (for many reasons), not a by-config plugin-dependency choice of a plugin, and --based on popularity of the dependant plugin autors and/or site-maintainer's config choices-- may affect dokuwiki's future.